modules/weko-records-ui/weko_records_ui/pdf.py

```diff
@@ -429,45 +445,57 @@ def get_oa_policy(activity_id):
             metadata_dict["affiliation"])
     ]
     metadata = '\n'.join(metadata_list)
-    metadata_lfnum = int(metadata.count('\n'))
-
-    for item in metadata_list:
-        metadata_lfnum += int(get_east_asian_width_count(item)
-                              ) // max_letters_num
 
     # Get url
     url = get_url(pid.pid_value)
-    url_lfnum = int(get_east_asian_width_count(url)) // max_letters_num
 
-    # Save top coordinate
-    top = pdf.y
     # Calculate x position of next cell
     offset = pdf.x + w1
+
+    # Save top coordinate
+    top = pdf.y
```

Comment:
I think this step is useless.

Reply:
It's used to get coordinates to add sell so it's necessary